Gems/AWSGameLift/Code/AWSGameLiftServer/Source/AWSGameLiftServerManager.cpp

```diff
@@ -452,9 +486,43 @@ namespace AWSGameLift
             return;
         }
 
-        AZ_TracePrintf(AWSGameLiftServerManagerName, "Initiating Amazon GameLift Server SDK ...");
-        Aws::GameLift::Server::InitSDKOutcome initOutcome = m_gameLiftServerSDKWrapper->InitSDK();
-        AZ_TracePrintf(AWSGameLiftServerManagerName, "InitSDK request against Amazon GameLift service is complete.");
+        Aws::GameLift::Server::Model::ServerParameters serverParameters;

```

Comment:
_Not feedback to address for this PR, but just including my thoughts maybe for later cleanup._

I feel a slight allergy to cvars that must be used during system component OnActivate() just because conceptually these aren't variables that are read and written to over the app's lifetime, but are instead executed once like an event. "sv_gameLiftEnabled" could be a console command "HostGameLiftServer" and "HostGameLiftAnywhereServer" with parameters containing whatever GLProper or GLAnywhere needs to start hosting. That would strongly tie all GLAnywhere parameters together since you'd be passing them all into the console command.

Reply:
Will keep this in mind for possible follow-ups!

Reply:
The nice thing about this particular approach though is all these cvar values can be dropped into a .cfg or a .setreg file that's deployed alongside the build and just autoloads at app startup.